orttraining/pytorch_frontend_examples/mnist_training.py

```diff
@@ -167,6 +168,19 @@ def main():
             transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])),
         batch_size=args.test_batch_size, shuffle=True, **kwargs)
 
+
+    comm = MPI.COMM_WORLD
```

Comment:
seems we are assuming this script only work for MPI distributed run, right?

Reply:
Also can run on single GPU, just use the "python mnist_training.py" command, and use mpi for 2gpu.